### PR TITLE
limit users for partners

### PR DESCRIPTION
**Pull request by smn**
When logged in as a manager, only download user stats for the partner.

### CODE REVIEW

File: casepro/statistics/tests.py [hunk 1]
```diff
@@ -337,6 +337,50 @@ def test_user_export(self):
...
+            [msg] = self.new_messages(d2, 1)
+            Case.get_or_open(self.nyaruka, self.user4, msg, 'summary', self.moh)
+
+        self.new_outgoing(self.user1, d1, 1)  # Jan 1st
+        self.new_outgoing(self.user3, d2, 1)  # Jan 15th
+
+        # logging in as user1, who's a manager for UNICEF not Nyaruka
+        self.login(self.user1)
+
+        response = self.url_post_json('unicef', url, {'type': 'U', 'after': "2016-01-01", 'before': "2016-01-31"})
```
Comment (smn) [inline casepro/statistics/tests.py]: This is returning with a redirect for login but I've not yet been able to figure out why.

**Comment by smn** [comment]
@jpaidoussi @rudigiesler @KaitCrawford ready for review

File: casepro/statistics/tests.py [hunk 2]
```diff
@@ -337,6 +337,50 @@ def test_user_export(self):
...
 
+    @override_settings(CELERY_ALWAYS_EAGER=True, CELERY_EAGER_PROPAGATES_EXCEPTIONS=True, BROKER_BACKEND='memory')
+    def test_manager_user_export(self):
+        url = reverse('statistics.dailycountexport_create')
+
+        tz = pytz.timezone("Africa/Kampala")
+        d1 = date(2016, 1, 1)
+        d2 = date(2016, 1, 15)
+
+        # Jan 1st, cases opened by User1 who's part of Nyaruka
```
Comment (jpaidoussi) [inline casepro/statistics/tests.py]: Do you mean User4 here?

Comment (smn) [inline casepro/statistics/tests.py]: Good grief. No, but it's a different silly mistake. Fixing now.

**Comment by smn** [comment]
@jpaidoussi test fixed again, ready for review.

**Comment by smn** [comment]
Putting this on hold, I don't think managers are to be given the option of downloading stats.

**Comment by smn** [comment]
Yeah this is going nowhere. 💣

**Pull request closed without merging**